backend/src/main/java/com/jobwait/persistence/PostgresController.java

```diff
@@ -0,0 +1,179 @@
+package com.jobwait.persistence;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.UUID;
+
+import com.jobwait.domain.Answers;
+import com.jobwait.domain.User;
+import com.jobwait.domain.ValidEducationLevel;
+import com.jobwait.domain.ValidWorkContract;
+import com.jobwait.domain.ValidWorkModel;
+import com.jobwait.persistence.adapters.PostgresUserAdapter;
+
+public class PostgresController extends PersistenceController {
+    private String jdbcUrl = "jdbc:postgresql://localhost:5432/mydatabase";
+    private String dbUser = "postgres";
+    private String dbPassword = "password";
+
+    private Connection getConnection() throws SQLException {
+        return DriverManager.getConnection(jdbcUrl, dbUser, dbPassword);
+    }
+
+    @Override
+    public User getUserFromAuthId(String authId) {
+        try {
+            Connection connection = getConnection();
+            PreparedStatement statement = connection.prepareStatement("SELECT * FROM users WHERE authhash = ?");
+            statement.setString(1, authId);
+            ResultSet resultSet = statement.executeQuery();
+            List<User> users = Util.resultSetRowsToAdaptedRows(resultSet, new PostgresUserAdapter());
+            return Util.assertSingleElement(users);
+        } catch (ElementNotFoundException e) {
+            throw new RuntimeException(String.format("Could not find user with authId: %s", authId));
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private class InsertAnswerTables {
+        private Connection connection;
+        private UUID userId;
+
+        InsertAnswerTables(Connection connection, UUID userId) {
+            this.connection = connection;
+            this.userId = userId;
+        };
+
+        // this should work for now cause each answer table is just (userid, response)
+        // frankly I don't think we'll ever need 2 columns for a user answer
+        private PreparedStatement createStatementForTable(String tableName) throws SQLException {
+            return connection.prepareStatement(
+                    String.format("INSERT INTO %s VALUES (?, ?) ON CONFLICT(userid) DO UPDATE SET response = EXCLUDED.response",
```

Comment:
at all worthwhile to consider a warning log? i guess we wont do that, still stuck in old ways of thinking we are working.... lol....

Reply:
A warning log for what? Unless I'm misunderstanding your comment, any error involved with this statement will throw an SQLException that will get printed out in some parent function, so we already sorta have a warning log, no?